README.md

```diff
@@ -1,66 +1,54 @@
-# API Compare Tool
+# Roave Backward Compatibility Check
```

Comment:
Overall, I aimed at a few things:

 1. cut to the point in the docs
 2. moved docs for the CLI command into the CLI command itself
 3. made the `roave` bit obnoxiously repetitive: I like throwing free time at OSS, but people can indeed type `roave` a few times.